### DISCUSSION TITLE
Turn off flip augmentation

### DISCUSSION THREAD

**Discussion by elitap** [MIC-DKFZ/nnUNet#551] [Q&A] [answered]
Hi Fabian,
what would be the fastest way to turn off the flip augmentation? I have a dataset with "symmetric" organs (optic nerves, parotid glands ....) where the left and right side are differntly labeled. I usually do not use a filp layer, or if I do so, I change the lable ids back to the one of its side. The plain nnUnet performs well, except for the symmetric organs, where the correct lable information is lost due to the flip. So for test purposes I would simply like to turn off the flip.

Thank you for the support and great work :-)

**Comment by elpequeno** [comment, marked as answer]
Hey elitap, 
I had a similar problem and I used "nnUNetTrainerV2_noMirroring.py". As far as I know, this does exactly what you want. 
I also realized that the 3D_cascade did not suffer from the "flip-effect". Maybe you can try if you see similar behavior.

Cheers, 
André

**Reply by elitap** [reply]
Hi André,
thank you for pointing me in the right direction :-). I just started the training, curious for the results.

Best,
eli
